frontend/src/pages/SearchMapPage.js

```diff
@@ -32,14 +43,13 @@ export default function SearchMapPage() {
     if(loadError) return "Error loading Maps";
     if(!isLoaded) return "Loading Maps";
     console.log(React.version)
+    console.log(selected)
+
     const mapContainerStyle = {
         width:"100vw",
         height: "100vh"
     };
```

Comment:
Can be moved outside the function, there is no need to redefine it on every update